File: cms_web/src/Services/Entities/ConsultImmuneEntity.ts
interface ConsultImmuneEntity {
  cr_immune_pk?: number;
  consult_req_pk?: string;
  vac_pk?: string;
  vac_desc?: string;
  vac_type?: string;
  date_given?: string | Date;
  next_dose?: string | Date;
  administered_by?: string;
  is_valid?: "y" | "n";
  encoded_at?: string | Date;
  encoder_pk?: string;
}

export interface ConsultImmuneTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultImmuneEntity>;
}

export default ConsultImmuneEntity;

File: cms_server/cms_server/Entities/ConsultImmuneEntity.cs
using System;

namespace cms_server.Entities
{
    public class ConsultImmuneEntity
    {
        public int? cr_immune_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string vac_pk { get; set; }
        public string vac_desc { get; set; }
        public string vac_type { get; set; }
        public DateTime? date_given { get; set; }
        public DateTime? next_dose { get; set; }
        public string administered_by { get; set; }
        public string is_valid { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}
